src/DynamoCoreWpf/ViewModels/ViewModelBase.cs

```diff
@@ -20,15 +20,6 @@ public bool IsDebugBuild
             }
         }
 
-        [JsonIgnore]
-        public bool IsPython2ObsoleteDebugModeEnabled
```

Comment:
why is this removed?

Reply:
That's a good question. I saw it was not being used. You could say removing it is a breaking change, but it was added a couple of weeks ago, probably by mistake.